Changes

```diff
@@ -156,6 +156,10 @@ Working version
 - #13740: Improve performance of Weak.find_aux
   (Josh Berdine, review by Gabriel Scherer)
 
+- #15049: Improve performance and type safety of Type.Id by using
```

Comment:
incorrect number

Reply:
Thanks, I've pushed a fix to trunk directly